site/content/3.13/data-science/arangographml/arangograph-ml.md

```diff
@@ -0,0 +1,183 @@
+---
+title: ArangoGraphML Web Interface
+menuTitle: ArangoGraphML Web Interface
+weight: 15
+description: >-
+ Enterprise-ready, graph-powered machine learning as a cloud service or self-managed
+aliases:
+  - getting-started-with-arangographml
+---
+Solve high-computational graph problems with Graph Machine Learning. Apply ML on a selected graph to predict connections, get better product recommendations, classify nodes, and perform node embeddings. Configure and run the whole machine learning flow entirely in the web interface.
```

Comment:
We only have node classification and embeddings available as immediate options. If we mention something like link predictions, we should at least outline how to achieve that.

Would also be good to have a more technical explanation here about how GraphML works (GraphSage, using depth 2 neighborhood, as mentioned in Slack team channel).

Please also add an overview over the process instead of immediately starting with project creation etc., users should first get an understanding of the hierarchy and steps involved.